modules/diskq/qdisk.c

```diff
@@ -210,6 +210,11 @@ _next_filename(QDisk *self, gchar *filename, gsize filename_len)
       return FALSE;
     }
 
+  gint fd = -1;
+  if (!_create_file(filename, &fd))
```

Comment:
`_create_file()` probably should not open the created file.

Reply:
Yes, you are right. Fixed :)